### ISSUE TITLE
Start protected Session causes error

### ISSUE THREAD

**Issue by christophersw** [zadam/trilium#350]
When attempting to enter a protected session: after putting in password and hitting enter I get the following error:

```
Trace: ERROR: Error executing transaction, executing rollback. Inner exception: Error: Option encryptedDataKeyIv doesn't exist
    at Object.getOption (/usr/src/app/src/services/options.js:7:15)Error
    at Object.transactional (/usr/src/app/src/services/sql.js:164:19)
    at cls.init (/usr/src/app/src/routes/routes.js:73:38)
    at /usr/src/app/node_modules/cls-hooked/context.js:118:13
    at Namespace.run (/usr/src/app/node_modules/cls-hooked/context.js:97:5)
    at Namespace.runAndReturn (/usr/src/app/node_modules/cls-hooked/context.js:117:8)
    at Object.init (/usr/src/app/src/services/cls.js:5:28)
    at router.(anonymous function) (/usr/src/app/src/routes/routes.js:68:38)
    at Layer.handle [as handle_request] (/usr/src/app/node_modules/express/lib/router/layer.js:95:5)
    at next (/usr/src/app/node_modules/express/lib/router/route.js:137:13)
    at checkApiAuth (/usr/src/app/src/services/auth.js:37:9)
    at Object.error (/usr/src/app/src/services/log.js:33:13)
    at Promise (/usr/src/app/src/services/sql.js:182:21)
Trace: ERROR: post /api/login/protected threw exception: Error: Option encryptedDataKeyIv doesn't exist
    at Object.getOption (/usr/src/app/src/services/options.js:7:15)
    at Object.error (/usr/src/app/src/services/log.js:33:13)
    at router.(anonymous function) (/usr/src/app/src/routes/routes.js:87:17)
Trace: ERROR: JS Error: Error when calling POST login/protected: 500 - Internal Server Error
    at Object.error (/usr/src/app/src/services/log.js:33:13)
    at WebSocket.ws.on.messageJson (/usr/src/app/src/services/messaging.js:31:21)
    at WebSocket.emit (events.js:182:13)
    at Receiver.receiverOnMessage (/usr/src/app/node_modules/ws/lib/websocket.js:741:20)
    at Receiver.emit (events.js:182:13)
    at Receiver.dataMessage (/usr/src/app/node_modules/ws/lib/receiver.js:417:14)
    at Receiver.getData (/usr/src/app/node_modules/ws/lib/receiver.js:347:17)
    at Receiver.startLoop (/usr/src/app/node_modules/ws/lib/receiver.js:133:22)
    at Receiver._write (/usr/src/app/node_modules/ws/lib/receiver.js:69:10)
    at doWrite (_stream_writable.js:410:12)
```

**Comment by zadam** [comment]
Did you downgrade from 0.28.0-beta to 0.27.X? The field it complains about ("encryptedDataKeyIv") existed in versions 0.27 and older.

**Comment by christophersw** [comment]
That's exactly what I did... How can I clean this up?

**Comment by christophersw** [comment]
Nevermind - I've got this figured out... 

Thanks for making such excellent software!

**Comment by zadam** [comment]
Your document has been migrated to 0.28 format so one way to fix it is to just upgrade to 0.28 again.

Another option is to restore the database from before the migration. Trilium automatically makes a backup for your document before every migration in trilium-data/backup/backup-before-migration.db so you can restore that into trilium-data/document.db and you can use it with 0.27 (of course you would lose all changes made in 0.28 though)
